README.md

```diff
@@ -1 +1,37 @@
-# html-css-training
\ No newline at end of file
+# HTML CSS Practice
+
+- 2024/02/16
+
+## Overview:
+
+- This document provides requirements, technical stack for HTML CSS Practice
+- Refer to the details design here: [Figma](<https://www.figma.com/file/36kEHivroepFyDPDFsNwjd/Damiun-Trip's-(Community)?type=design&node-id=0-1&mode=design&t=yE5y31zDjW8FPgeH-0&fbclid=IwAR21Xc-sYMmXajBrOQ2kTU9ncuOT_bRRyqkq2yk0K4HZMwne9LiVWwAFw-s>)
+
+## Timeline:
+
+- 42 hours (2024/02/16 - 2024/02/23)
+
+## Team size:
+
+- 1 developer
+
+## Technical:
+
+- HTML5/CSS3
+- Vercel
+- Parcel
+
+## Task Management:
+
+- [Trello](https://www.google.com/url?q=https://trello.com/b/IyxM90wS/html-css-training&sa=D&source=docs&ust=1708082038564227&usg=AOvVaw1rnxaxLxsZYYj0QvFT4VUu)
+
+## Requirements:
+
+- Apply mobile first for practice
+- Apply CSS guideline
+- Apply Flexbox and Grid
+- Responsive (Mobile, Tablet, Desktop)
+- Cross-browser testing (MS Edge, Chrome, Firefox latest version)
+  - MS Edge: 44.17763.1.0
+  - Chrome: 121.0.6167.185
+  - Firefox: 122.0.1
```

Comment:
Should add a `Getting Start` section to guild run your practice. You can check the sample below:


## Getting Start:
- Clone the repository
```git clone https://github.com/JennyDoJD/html-css-training.git```
- Move to the cloned repository folder
```cd html-css-training```
- Checkout to branch `html-css-practice`
```git checkout feature/html-css-practice```
- Move to folder 
```cd .... ```
- Install packages
```npm install```
- Run the server
```npm start```
- Open your web browser and navigate to http://localhost:1234/